src/admin-views/tribe-options-general.php

```diff
@@ -270,6 +269,12 @@
 		return $fields;
 	} );
 } else {
+	if ( tribe_is_truthy( tribe_get_option( 'tribeDisableTribeBar', false ) ) ) {
+		$filter_activation[ 'attributes' ] = [ 'disabled' => 'disabled' ];
+		$filter_activation[ 'class' ]      = 'tribe-fieldset-disabled';
+		$filter_activation[ 'tooltip' ]    = esc_html__( 'This option is disabled when "Disable the Event Search Bar" is checked on the Display settings tab.', 'the-events-calendar' );
```

Comment:
:no_entry_sign: Array keys must NOT be surrounded by spaces if they only contain a string or an integer.

<sub>hash: 1bf9f9784c3c313f2ae6edd86db5c12d</sub>